synchronizer/connectors/jira.py

```diff
@@ -16,7 +18,15 @@ def __init__(self, **kwargs):
         )
 
     def _api(self, method, endpoint, data=None, params=None):
-        response = requests.request(
+        # Apply delays between attempts to connection to
+        # jira server in case of maximum requests quota
+        session = requests.Session()
+        retry = Retry(connect=3, backoff_factor=0.5)
+        adapter = HTTPAdapter(max_retries=retry)
+
+        session.mount('https://', adapter)
```

Comment:
I think that we should to move this to `__init__`. This makes no sense to re-create session for each request to API.